src/modules/login/containers/LoginPage.js

```diff
@@ -21,7 +21,7 @@ export class LoginPage extends Component {
   componentWillReceiveProps(nextProps) {
     if (nextProps.user != undefined) {
       if (typeof nextProps.user.apiToken !== "undefined") {
```

Comment:
Is it that you want to check if the `apiToken` variable exists or the `value` of the `apiToken` variable? What are you trying to achieve?

Reply:
@Jeroen-0606